NEWS

```diff
@@ -39,6 +39,27 @@ New Functionality
 - SMB2 packets containing multiple PDUs now correctly parse all of the headers,
   instead of just the first one and ignoring the rest.
 
+- The new built-in functions ``lookup_conn_analyzer_id_by_name()`` and
+  ``lookup_conn_analyzer_id_by_tag()`` retrieve the numeric identifier of an
+  analyzer associated with a connection. This enables the use of the
+  ``disable_analyzer()`` BiF outside of the analyzer confirmation/violation
+  events that have so far been the only providers of those identifiers. For
+  example, this allows the suppression of an analyzer from the outset for
+  specific connections:
+
+	event connection_established(c: connection):
+		{
+		if ( no_http_for_this_conn_wanted(c) )
+			{
+			local aid = lookup_conn_analyzer_id_by_tag(c$id, Analyzer::ANALYZER_HTTP);
+			if ( aid > 0 )
+				disable_analyzer(c$id, aid, T, T);
+			}
+		}
+
+  The ``lookup_conn_analyzer_id_by_name()`` BiF does the same, but looks up the
```

Comment:
Can't this one just be done via composition, using the first one plus the existing `Analyzer::get_tag()`?

Reply:
True! Thank you for flagging that. I'll drop that one.

Reply:
I've renamed the one BiF remaining to `lookup_connection_analyzer_id()`, for symmetry with the existing `lookup_connection()`.